resolwe/storage/management/commands/run_storage_cleanup.py

```diff
@@ -0,0 +1,38 @@
+""".. Ignore pydocstyle D400.
+
+============================
+Command: run_storage_cleanup
```

Comment:
Use Django's style of command names - without underscores, like makemigrations, startapp,...